playwright-e2e/pages/start-application.ts

```diff
@@ -54,6 +56,8 @@ export class StartApplication {
     this.welshLanguageRequirements = page.getByRole('link', { name: 'Welsh language requirements' });
     this.welshLanguageReqFinished = page.locator('p:has(a[text()="Welsh language requirements"]) > img[title="Finished"]');
```

Comment:
Do the other locators need to be adjusted too? can see some have () removed but not others?